x-pack/platform/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/api_key.ts

```diff
@@ -124,7 +123,7 @@ export default function apiKeyBackfillTests({ getService }: FtrProviderContext)
     }
 
     it('should wait to invalidate API key until backfill for rule is complete', async () => {
-      const start = moment().utc().startOf('day').subtract(13, 'days').toISOString();
```

Comment:
reducing the time span we're scheduling the backfill for. for this test, we have to wait for the entire backfill to finish to check that the API key gets deleted afterward, we're reducing the number of backfill jobs we're waiting for from 18 to 8, decreasing the time the test takes to run.